packages/test-interface-criteria/src/criteria/criteriaUtils.js

```diff
@@ -65,7 +65,7 @@ function getRemainingCapacity(opportunity) {
  */
 function mustBeWithinBookingWindow(offer, opportunity) {
   if (!offer || !offer.validFromBeforeStartDate) {
-    return false;
+    return null; // Required for validation step
```

Comment:
Sorry, what's this for?

Reply:
Without it, validation fails, complaining that the string isn't a valid ISO offset. It would also work fine with `false`, but I wanted to make it explicit that it can't be within or outside the booking window if there is no booking window.